packages/adp-tooling/src/types.ts

```diff
@@ -35,21 +35,41 @@ export interface AdpWriterConfig {
     target: AbapTarget;
     ui5?: {
         minVersion?: string;
+        ui5Version?: string;
+        ui5EndpointUrl?: string;
     };
     package?: {
         name?: string;
         description?: string;
     };
+    flp?: {
+        flpTitle?: string;
+        flpSubtitle?: string;
```

Comment:
Since these are under `flp` maybe we can omit this prefix `flp`? So it becomes this:
```suggestion
        title?: string;
        subtitle?: string;
```

Reply:
Done